config/locales/client.en.yml

```diff
@@ -27,6 +27,8 @@ en:
       assigned: "Assigned"
       group_everyone: "Everyone"
       assigned_to: "Assigned to"
+      topic_to: "topic to"
```

Comment:
Paging doctor @gschlager - as I understand, partial strings like that aren't nicely translatable and there should be one (or more) complete string that's a full message?

Reply:
I agree, this one might be tricky enough to require the more advanced Message Format string?

Reply:
Hmm, this is hard to solve correctly unless widgets allow something like `htmlSafe` so that we can put the whole string into the locale file. Correct me if I'm wrong, but I don't think we have something like that for widgets.

Aren't we planning to replace widgets in the near future? I guess in that case it's fine to accept this PR as is. But I'd recommend adding comments to the new strings in the locale file so that translators know what comes before and after those strings in the UI.